### DISCUSSION TITLE
[Solved] Surface Pro 5 - Pairing problem with some devices

### DISCUSSION THREAD

**Discussion by LittleLion** [sebanc/brunch#2049] [Q&A]
Hello everyone,

I have a SP5, and i install the ChromeOs via brunch, and that is awesome, but i'm getting some problems, pairing some type of devices, like mouses, xbox controllers, etc, but my headphones it pairs well and working great. I detect the device, put to pair, and stays a long time saying "connecting" but in the end gave an error. 
Do you have any idea how to solve it?

Thanks,

**Comment by yasurok** [comment]
Hi, LittleLion. I also run Brunch on my Surface Pro (5) LTE Advanced. I'm sorry that I cannot help you with your question, but let me ask you this. Is your touch screen working? If so, which version of ChromeOS are you running?

I tried to solicit info [with this post](https://github.com/sebanc/brunch/discussions/1988), but unfortunately, it did not get much love.

Thanks!

**Reply by LittleLion** [reply]
Hello Yasurok, 

Regarding your question, yes the touch screen works perfect you just need to activate in the kernel the IPTS flag.
I run Rammus R121, and actually when i have the problems with the bluetooth i was running Brunch r121, and i just upated to Brunch r122 and my problems were fixed, with this latest version.

If you have any more doubts regarding the SP5 with brunch send it ;)

Best regards,

**Reply by yasurok** [reply]
That's great news. Thanks for letting me know. Do you happen to have an MPP pen? If you do, does it work with Brunch r122?

**Reply by LittleLion** [reply]
No problem. I don't have it , sorry.

**Reply by yasurok** [reply]
That's quite all right. Thanks all the same!
